### DISCUSSION TITLE
NO OFFLINE MODE??

### DISCUSSION THREAD

**Discussion by alexpi4** [QuestCraftPlusPlus/QuestCraft#213] [Q&A] [answered]
NO OFFLINE MODE??

**Comment by D2Lx0wse** [comment]
Microsoft fault. Complain with them, maybe they'll stop asking people with offline mode in their launchers to pay an extremely big fine

**Comment by WaffleDevs** [comment, marked as answer]
Offline mode is often used to pirate Minecraft. Thankfully, its not that hard to make enough money to get a Minecraft account.

**Comment by jakershaker1** [comment]
offline mode was removed so that questcraft could be ready for release on applab. Offline mode allowed for the breaking of the mojang TOS and if it stayed QC had no chance for an applab release.

**Comment by Flamgop** [comment]
Locking conversation as this is intentional and needs no further explanation
